app/cdap/components/shared/ConfigurationGroup/PropertyRow/LockedTooltip.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+import LockIcon from '@material-ui/icons/Lock';
+import PropertyRowTooltip from './PropertyRowTooltip';
+
+const styles = (theme): StyleRules => {
```

Comment:
i know this is copied from above but should we replace this with styled components style css?

Reply:
Converted for this and `DescriptionTooltip`. `PropertyRowTooltip` broke with styled components (some theming issue) and I don't have the bandwidth to troubleshoot it right now.